### DISCUSSION TITLE
Theming for the home (~) directory (Revisited for v0.3)

### DISCUSSION THREAD

**Discussion by fdnt7** [sxyazi/yazi#1469] [1. Q&A] [answered]
Sorry for the semi-necropost, but as of Yazi v0.3, [this solution](https://github.com/sxyazi/yazi/discussions/1077#discussioncomment-9555940) which was on a discussion I also started has stopped working as the `File` variable was removed. How could this still be done with the updated version of Yazi?

**Comment by sxyazi** [comment, marked as answer]
I've updated my original comment
